core/src/es/event.rs

```diff
@@ -261,6 +262,29 @@ where
     }
 }
 
+#[cfg(feature = "stored")]
+/// [`Event`] that can be stored in a repository. Holds metadata separately from
+/// the [`Event`] itself.
+#[derive(Clone, Debug, Default, serde::Deserialize, serde::Serialize)]
+pub struct Stored<'a, Ev: Revisable> {
+    /// Name of the [`Event`].
+    #[serde(rename = "n")]
+    pub name: Cow<'a, str>,
```

Comment:
@50U10FCA7 let's use `Raw` from #4. This is not restricted to be used for storing only. Feature gating is not required.

Also, `serde` renaming shoud be removed. If one needs customizing `serde`serializtion, he always may either wrap `Raw` with a custom type defining `serde`serialization strategy, or even use it's own type along with `From`/`Into` conversions.

`arcane` library should not dicate any specific format for serialization or storing.

Reply:
Furthermore, the `revision` here is mandatory at the moment. But the library user may want to omit using revisions for event. We should think how to follow this optionality naturally.